tfhub_dev/assets/intel/intel.md

```diff
@@ -0,0 +1,12 @@
+# Publisher intel
```

Comment:
Can you update the file location of all files in this PR to be under `assets/docs`?